packages/playground/src/components/project-settings/form-member-edit-dialog.tsx

```diff
@@ -125,8 +125,13 @@ export const FormEditMemberDialog: React.FC<FormEditDialogProps> = ({
                       </Button>
                     </>
                   ) : (
-                    <Button variant="ghost" type="button" size="sm" theme="success" className="pointer-events-none">
-                      Saved&nbsp;&nbsp;
+                    <Button
+                      variant="ghost"
+                      type="button"
+                      size="sm"
+                      theme="success"
+                      className="flex gap-2 pointer-events-none h-9">
```

Comment:
Do we need this custom class for the button? It might not remain consistent across.

Reply:
Yes, please see what I deleted and the gap can show the same style instead of using &nbsp

Reply:
probably nitpicking but imho we should just use the non-breaking space in html just so it remains consistent especially if it does the same thing.